src/components/Requirements/IncompleteSelfCheck.vue

```diff
@@ -146,6 +146,8 @@ export default Vue.extend({
   }
   &-content {
     z-index: 2;
+    position: absolute;
+    width: 88%;
```

Comment:
🤔

What's the 12% percent? Maybe we should use flexbox?

Reply:
This magic number already existed in the codebase (I actually removed this line in #312 when I was struggling with the requirements styling), so this PR just added it back when I realized I could fix the issues in a better way that also worked for mobile. I can still look into fixing it with flex or something if you think that is a good idea.